metricbeat/module/logstash/test_logstash.py

```diff
@@ -20,8 +22,6 @@ def test_node(self):
         """
         logstash node metricset test
         """
-        unittest.skip('Skipping this test to check documented fields. We will unskip once we know which fields can be deleted')
```

Comment:
Note that these tests (and the class) already have a top-level skip, so just removing the redundant lines here (especially since if we use this approach the test looks like a pass).